graylog2-server/src/main/java/org/graylog2/plugin/inputs/Extractor.java

```diff
@@ -159,115 +164,122 @@ public Extractor(MetricRegistry metricRegistry,
             this.regexConditionPattern = Pattern.compile(conditionValue, Pattern.DOTALL);
         }
 
-        this.totalTimerName = name(getClass(), getType().toString().toLowerCase(Locale.ENGLISH), getId(), "executionTime");
-        this.converterTimerName = name(getClass(), getType().toString().toLowerCase(Locale.ENGLISH), getId(), "converterExecutionTime");
+        final String metricsPrefix = name(getClass(), getType().toString().toLowerCase(Locale.ENGLISH), getId());
+        this.conditionHitsCounterName = name(metricsPrefix, "conditionHits");
+        this.conditionMissesCounterName = name(metricsPrefix, "conditionMisses");
+        this.conditionTimerName = name(metricsPrefix, "conditionTime");
+        this.executionTimerName = name(metricsPrefix, "executionTime");
+        this.converterTimerName = name(metricsPrefix, "converterExecutionTime");
+        this.completeTimerName = name(metricsPrefix, "completeExecutionTime");
```

Comment:
Can we also pre-create these timers like the counters below?
That should save another hash lookup and instanceOf check in `metricRegister.timer`, right?